<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigWidget</class>
 <widget class="QWidget" name="ConfigWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>363</width>
    <height>236</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Build Directory:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KUrlRequester" name="buildDir">
       <property name="toolTip">
        <string>Select the directory where the tools are being executed in.
If this is the same as the source directory you can leave it empty.</string>
       </property>
       <property name="mode">
        <set>KFile::Directory|KFile::ExistingOnly|KFile::LocalOnly</set>
       </property>
       <property name="clickMessage">
        <string>The directory where the tools are executed</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Build Tools</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Action:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="KComboBox" name="buildAction"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Enable:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QCheckBox" name="enableAction">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="execLabel">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Executable:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="KUrlRequester" name="actionExecutable">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>This is the executable that should be executed when selecting the action from the Project menu</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="argLabel">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Arguments:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="actionArguments">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>The arguments that the tool needs to execute properly.</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="envLabel">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Environment:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="KDevelop::EnvironmentSelectionWidget" name="actionEnvironment">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>Choose a pre-configured environment for your tool</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>76</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KUrlRequester</class>
   <extends>QFrame</extends>
   <header>kurlrequester.h</header>
  </customwidget>
  <customwidget>
   <class>KDevelop::EnvironmentSelectionWidget</class>
   <extends>QComboBox</extends>
   <header>util/environmentselectionwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>enableAction</sender>
   <signal>toggled(bool)</signal>
   <receiver>actionExecutable</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>169</x>
     <y>100</y>
    </hint>
    <hint type="destinationlabel">
     <x>168</x>
     <y>117</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>enableAction</sender>
   <signal>toggled(bool)</signal>
   <receiver>actionArguments</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>100</x>
     <y>94</y>
    </hint>
    <hint type="destinationlabel">
     <x>109</x>
     <y>142</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>enableAction</sender>
   <signal>toggled(bool)</signal>
   <receiver>actionEnvironment</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>99</x>
     <y>96</y>
    </hint>
    <hint type="destinationlabel">
     <x>103</x>
     <y>169</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>enableAction</sender>
   <signal>toggled(bool)</signal>
   <receiver>execLabel</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>128</x>
     <y>92</y>
    </hint>
    <hint type="destinationlabel">
     <x>51</x>
     <y>113</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>enableAction</sender>
   <signal>toggled(bool)</signal>
   <receiver>argLabel</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>115</x>
     <y>86</y>
    </hint>
    <hint type="destinationlabel">
     <x>35</x>
     <y>138</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>enableAction</sender>
   <signal>toggled(bool)</signal>
   <receiver>envLabel</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>121</x>
     <y>100</y>
    </hint>
    <hint type="destinationlabel">
     <x>34</x>
     <y>176</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
